draft-ietf-ppm-dap.md

```diff
@@ -1217,15 +1222,14 @@ conveyed from the Leader to the Helper when initializing aggregation jobs
 
 ## Resource URLs
 
-DAP is defined in terms of "resources", such as reports ({{upload-flow}}),
+DAP is defined in terms of HTTP resources, such as reports ({{upload-flow}}),
```

Comment:
Avoid scare quotes and make explicit reference to HTTP, which has a strong concept of a [resource](https://datatracker.ietf.org/doc/html/rfc9110#section-3.1).